src/tests/initial-state.test.js

```diff
@@ -0,0 +1,38 @@
+import { screen } from '@testing-library/react'
+import { setupTest } from './setupTest'
+
+const mockWordToGuess = 'hello'
+
+describe('Initial state', () => {
+  test('shows correct message', async () => {
+    await setupTest(mockWordToGuess)
+    expect(screen.getByText('Please enter your guess and submit')).toBeDefined()
+  })
+
+  test('shows underscores for every letter', async () => {
+    await setupTest(mockWordToGuess)
+    expect(
+      screen.getByText('_ '.repeat(mockWordToGuess.length).trim()),
```

Comment:
I would actually type out full expected value here. It would be easier to see the expectation without needing the compute the algorithm in the head or actually run the test what the app is supposed to do with the word.